src/index.ts

```diff
@@ -21,9 +21,9 @@ export = class OutputCsv {
   public async serve(req: Request, res: Response) {
     res.setHeader('Content-Type', 'text/csv');
     try {
-      const csvTemplate = _.get(req, 'res.locals.csvTemplate') as Record<string, any>;
-      const csvTemplateTransforms = _.get(req, 'app.locals.csvTemplateTransforms') as TransformsList;
-      const csvFileName = _.get(req, 'app.locals.csvFileName', 'output');
+      const csvTemplate = _.get(req, 'app.locals.csvTemplate') || _.get(req, 'res.locals.csvTemplate') as Record<string, any>;
```

Comment:
Wait, shouldn't the `req.res.locals.csvTemplate` override what we have defined at the app-level in `app.locals.csvTemplate`?

Reply:
That's a good point! I was assuming either one of them to be defined at a time but it makes sense when both are defined, request-level option should be be prioritized instead of application-level options. I have re-adjusted the order accordingly.